Parsers/ASimAuditEvent/Parsers/vimAuditEventBarracudaWAF.yaml

```diff
@@ -192,103 +192,7 @@ ParserQuery: |
                 ManagementGroupName,
```

Comment:
please bring up the time filters, keep just after where not(disabled)

Reply:
done